WardrobeCloset/Controllers/OutfitsController.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.Entity;
+using System.Linq;
+using System.Net;
+using System.Web;
+using System.Web.Mvc;
+using WardrobeCloset.Models;
+
+namespace WardrobeCloset.Controllers
+{
+    public class OutfitsController : Controller
```

Comment:
I've noticed that many of your controllers share the same database, and some of the same code bits. I'm not sure exactly how this would look in C#, but I'm sure it's possible to create base controller class which would contain the common bits (like the private db, some of the common lines that I mentioned above, etc). Then you could easily re-use in each subclass.

I'm not sure if you've done anything with subclassing your coursework; if so, great, if not, don't sweat it! Of course your code will work as is just fine.